templates/impala/header_title.html

```diff
@@ -1,7 +1,7 @@
 <h1 class="site-title">
 {% macro heading(app, text, icon) -%}
 <a href="{{ url('home') }}" title="{{ _('Return to the {0} Add-ons homepage')|f(request.APP.pretty) }}">
-  <img alt="{{ APP.pretty }}" src="{{ static('img/app-icons/med/' + icon + '.png')|safe }}">
+  <img alt="" src="{{ static('img/icons/firefox.png')|safe }}">
```

Comment:
You should keep an alt attribute;, it's important for accessibility.

Also, make sure to set the size of the image so you can use a retina file so us retina users don't suffer :wink: